content/cybersecurity/concepts/penetration-testing/penetration-testing.md

```diff
@@ -0,0 +1,24 @@
+---
+Title: 'Penetration Testing'
+Description: 'Penetration Testing, also referred to as pen testing, involves breaking into a system to better identify cybersecurity weaknesses so that the security posture can be improved.'
```

Comment:
```suggestion
Description: 'Penetration testing, also referred to as pen testing, involves attempting to break into a system to better identify cybersecurity weaknesses so that the security posture can be improved.'
```